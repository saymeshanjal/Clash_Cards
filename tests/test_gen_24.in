11
10 19 2
4 334 4
4 356 5
6 182 3
12 248 1
2 337 1
12 252 1
16 395 4
4 145 3
8 47 3
14 315 3
