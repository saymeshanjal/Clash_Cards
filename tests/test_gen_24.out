8 33 211
0 5 1
0 6 0
6 14 0
462 462 0
1 1 0
462 462 0
3809 5406 3655
1 5 0
27 43 0
1652 1780 0
